changelogs/v3-upgrade-guide.md

```diff
@@ -48,7 +49,7 @@
 
 ## Deprecation
 
-### Deprecating ImplicitMSALAuthenticationProvider 
-* Use of ImplicitMSALAuthenticationProvider, that is,using the implicit authorization flow is not recommended any more.
-* Use the TokenCredentialAuthenticationProvider with azure/identity library or a CustomAuthenticationProvider with an auth library of your choice instead.
+### Deprecating `ImplicitMSALAuthenticationProvider` 
+* Use of `ImplicitMSALAuthenticationProvider`, that is,using the implicit authorization flow is not recommended any more. [OAuth 2.0 Implicit Grant](https://oauth.net/2/grant-types/implicit/).
+* Alternatives to `ImplicitMSALAuthenticationProvider` - TokenCredentialAuthenticationProvider with azure/identity library or a CustomAuthenticationProvider with an auth library of your choice.
```

Comment:
can we add which credential they should use in combination with the TokenCredentialProvider? and maybe a link to this? https://docs.microsoft.com/en-us/graph/sdks/choose-authentication-providers